plugin/trino-hive/src/main/java/io/trino/plugin/hive/DefaultHiveTableRedirectionsProvider.java

```diff
@@ -18,15 +18,30 @@
 import io.trino.spi.connector.ConnectorSession;
 
 import java.util.Optional;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
 
+import static io.trino.plugin.hive.HiveSessionProperties.getDeltaLakeCatalogName;
 import static io.trino.plugin.hive.HiveSessionProperties.getIcebergCatalogName;
+import static io.trino.plugin.hive.util.HiveUtil.isDeltaLakeTable;
 import static io.trino.plugin.hive.util.HiveUtil.isIcebergTable;
 
 public class DefaultHiveTableRedirectionsProvider
         implements HiveTableRedirectionsProvider
 {
     @Override
     public Optional<CatalogSchemaTableName> redirectTable(ConnectorSession session, Table table)
+    {
```

Comment:
as a followup, we should remove `DefaultHiveTableRedirectionsProvider` and `HiveTableRedirectionsProvider`

Reply:
I already have it on my todo list.